### PR TITLE
typo

### PR DESCRIPTION
**Pull request by github-john-doe**

### CODE REVIEW

**Comment by github-john-doe** [comment]
@StephenBonikowsky

File: src/System.Private.ServiceModel/src/System/ServiceModel/Channels/Connection.cs [hunk 1]
```diff
@@ -614,10 +614,10 @@ public void EndWrite()
             }
         }
 
-        private void OnWrite(Task antecedant, Object state)
+        private void OnWrite(Task antecedent, Object state)
         {
             Contract.Requires(_writeResult == null, "StreamConnection: OnWrite called twice.");
-            _writeResult = antecedant;
+            _writeResult = antecedent;
```
Comment (StephenBonikowsky) [inline src/System.Private.ServiceModel/src/System/ServiceModel/Channels/Connection.cs]: Two more instances of this typo on lines 724 and 727, can you fix those as well?

Comment (github-john-doe) [inline src/System.Private.ServiceModel/src/System/ServiceModel/Channels/Connection.cs]: Fixed

**Comment by StephenBonikowsky** [comment]
LGTM

**Comment by github-john-doe** [comment]
Thanks!

**Pull request merged**